libsplinter/Cargo.toml

```diff
@@ -151,7 +151,7 @@ https-bind = ["actix-web/ssl"]
 memory = ["sqlite"]
 oauth = ["biome", "oauth2", "reqwest", "rest-api"]
 postgres = ["diesel/postgres", "diesel_migrations"]
-registry = []
+registry = ["reqwest"]
```

Comment:
Replace this with something like:

registry = []
registry-client = ["registry"]
registry-client-reqwest = ["registry-client", "reqwest"]

Reply:
Fixed